### PR TITLE
New 3D printed parts 

### LINKED ISSUE #177: Add new 3D printed parts
### Describe the solution you'd like

* [x] Add new controller case files.
* [x] Add new controller case lid files.
* [x] Add GPS base station antenna stand adapter. 
* [x] Update edited anemometer case
* [x] Update relevant readme.

### PR DESCRIPTION
**Pull request by jamie-54**
## Main Changes

Added the parts that have been 3D printed recently.

## Associated Issues

- Closes #177 

## Tests

Controller case and lid fit the F450 frame
Base station antenna adapter fits the camera tripod

### CODE REVIEW

File: design/Parts_for_3D_printing/GPS_and_antenna_parts/Base_station_adapter.png
[the remark this answers is not in the document]
Comment (jamie-54) [reply design/Parts_for_3D_printing/GPS_and_antenna_parts/Base_station_adapter.png:1]: Yea should be the gap is 42mm by 92mm

[the remark this answers is not in the document]
Comment (jamie-54) [reply design/Parts_for_3D_printing/GPS_and_antenna_parts/Base_station_adapter.png:1]: I've sent for it to be printed so we can check after

**Review by Runway27 — APPROVED** [review]

**Comment by jamie-54** [comment]
The Base station antenna adapter has been printed and fits so I think we can merge this PR now. What's your thoughts?

**Comment by alphaville** [comment]
Yes, @jamie-54, let's merge this

**Pull request merged**